packages/nestjs/src/index.ts

```
@@ -0,0 +1,318 @@
+} from '@graphql-tools/utils';
+import { Injectable, Logger as NestLogger } from '@nestjs/common';
+import {
+  AbstractGraphQLDriver,
+  GqlSubscriptionService,
+  GraphQLWsSubscriptionsConfig,
+  type GqlModuleOptions,
+  type SubscriptionConfig,
+} from '@nestjs/graphql';
+import type { FastifyReply, FastifyRequest } from 'fastify';
+
+export type HiveGatewayDriverConfig<TContext extends Record<string, any> = {}> =
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)